### PR TITLE
[Merged by Bors] - feat: bump version to `0.2.1`

### PR DESCRIPTION
**Pull request by EstebanBorai**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@ jobs:
     with:
       package-name: http-sink
       branch: ${{ github.event.inputs.branch }}
-      fail-fast: false  
+      fail-fast: false
+      readme: '../../README.md'
```
Comment (morenol) [inline .github/workflows/publish.yml:24]: Is this path relative to the `Connector.toml`? or relative to where the command is running?

Comment (EstebanBorai) [reply .github/workflows/publish.yml:24]: To command is running! Yeah I think it should be `readme: './README.md'` on first sight I assumed the action to publish connectors changed directory to package's dir (being `crates/http-sink`) so I did `../../` but actually is not required. Fixing it..

**Review by morenol — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -112,16 +118,18 @@ jobs:
...
-        run: fluvio install cdk
+        run: fluvio install cdk --develop
 
       - name: Install Rust ${{ matrix.rust }}
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.rust }}
-          target: wasm32-unknown-unknown
+          target:
+            - wasm32-unknown-unknown
```
Comment (morenol) [inline .github/workflows/ci.yml:128]: why do we need this target?

Comment (EstebanBorai) [reply .github/workflows/ci.yml:128]: Actually I think we dont, I could remove it of course!

Comment (EstebanBorai) [inline .github/workflows/ci.yml:128]:
```suggestion
```

**Review by EstebanBorai — COMMENTED** [review]
Remove wasm toolchain as not being in use

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -112,16 +118,16 @@ jobs:
...
       - name: Install Fluvio CDK
-        run: fluvio install cdk
+        run: fluvio install cdk --develop
 
       - name: Install Rust ${{ matrix.rust }}
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.rust }}
-          target: wasm32-unknown-unknown
+          target: x86_64-unknown-linux-musl
```
Comment (morenol) [inline .github/workflows/ci.yml:127]: we should use here `${{ matrix.rust-target }}`

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -112,16 +118,16 @@ jobs:
...
 
       - name: Build Connector
-        run: cdk build -p http-sink
+        run: cdk build -p http-sink --target x86_64-unknown-linux-musl
```
Comment (morenol) [inline .github/workflows/ci.yml:130]: we should use here ${{ matrix.rust-target }}

File: README.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ HTTP Sink is configured using a YAML file:
 # config-example.yaml
 apiVersion: 0.1.0
 meta:
-  version: 0.1.0
+  version: 0.2.1
```
Comment (morenol) [inline README.md:19]: could you update the other places of the readme with `version: 0.1.0`?

**Review by morenol — APPROVED** [review]

**Pull request closed without merging**